build.sbt

```diff
@@ -0,0 +1,58 @@
+import ReleaseTransformations._
+
+name := "commercetools-sunrise-email"
+
+organization := "io.commercetools.sunrise"
+
+lazy val `commercetools-sunrise-email` = (project in file("."))
+  .configs(IntegrationTest)
+  .settings(commonSettings ++ commonTestSettings : _*)
+
+lazy val commonSettings = releaseSettings ++ Seq (
+  scalaVersion := "2.11.7",
+  javacOptions ++= Seq("-source", "1.8", "-target", "1.8")
+)
+
```

Comment:
I could, but it will be empty now. What kind of dependencies do you have in mind?